realm-jni/build.gradle

```diff
@@ -87,15 +87,34 @@ if (project.forceDownloadCore instanceof String) {
     ext.forceDownloadCore = Boolean.parseBoolean(project.forceDownloadCore)
 }
 
+def getNdkDir() {
+    def ndkDir = null
+    if (project.ext.properties.containsKey('ndk.dir')) {
+        ndkDir = project.ext['ndk.dir']
+    } else if (System.getenv('ANDROID_NDK')) {
```

Comment:
IMO it is confusing to support both. Just pick one of them. You can battle it out with @emanuelez which one you prefer, personally I would vote for `ANDROID_NDK` but I don't have a strong preference.